src/main/java/com/neuvector/DockerRunCommandBuilder.java

```diff
@@ -52,8 +44,18 @@ DockerRunCommandBuilder withEnvironment(String environmentVariable) {
         return this;
     }
 
-    String[] buildForImage(String image) {
+    DockerRunCommandBuilder withImage(String image) {
         cmdList.add(image);
-        return cmdList.toArray(new String[0]);
+        return this;
+    }
+
+    DockerRunCommandBuilder withCommand(String command) {
```

Comment:
Instead of running the default entrypoint of the container, this allows us to run multiple commands which in turns allows us to `cat scan_result.json` after we run `monitor` (the default entry point).